### PR TITLE
add missing namespace to mongodb notes

### PR DESCRIPTION
**Pull request by eguven**
**What this PR does / why we need it**:

Adds namespace to the `kubectl run` command in NOTES.txt for connecting to database.

### CODE REVIEW

File: stable/mongodb/templates/NOTES.txt [hunk 1]
```diff
@@ -40,7 +40,7 @@ To get the password for "{{ .Values.mongodbUsername }}" run:
 
 To connect to your database run the following command:
 
-    kubectl run {{ template "mongodb.fullname" . }}-client --rm --tty -i --image bitnami/mongodb --command -- mongo admin --host {{ template "mongodb.fullname" . }} {{- if .Values.usePassword }} -u root -p $MONGODB_ROOT_PASSWORD{{- end }}
+    kubectl run --namespace {{ .Release.Namespace }} {{ template "mongodb.fullname" . }}-client --rm --tty -i --image bitnami/mongodb --command -- mongo admin --host {{ template "mongodb.fullname" . }} {{- if .Values.usePassword }} -u root -p $MONGODB_ROOT_PASSWORD{{- end }}
```
Comment (davidkarlsen) [inline stable/mongodb/templates/NOTES.txt:43]: can you bump the chart-version too? TIA

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Comment by tompizmor** [comment]
Yes please, bump the chart version so the circle-ci job pass.

https://circleci.com/gh/helm/charts/21563?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Review by tompizmor — CHANGES_REQUESTED** [review]
Yes please, bump the chart version so the circle-ci job pass.

https://circleci.com/gh/helm/charts/21563?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Pull request merged**